components/Projects/ProjectsPage/ProjectDevelopmentModal/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { DarkContext } from 'helpers/user'
+import React, { useContext } from 'react'
+import { Modal, ModalHeader, ModalBody, Button } from 'reactstrap'
+
+const ProjectDevelopmentModal = ({
+  isOpen,
+  closeModal
+}: {
+  isOpen: Boolean
+  closeModal: Function
+}) => {
+  const isDark = useContext(DarkContext)
+
+  const closeBtn = (
```

Comment:
Isn't memoised

Reply:
I don't think I'm going to worry about memoising at this point unless you want to walk me through it